docbook/Developers_Guide/en-US/Events_Reference_Output.xml

```diff
@@ -240,6 +240,34 @@ array( '&lt;a href="' . plugin_page( 'mypage' ) . '">' . plugin_lang_get( 'mypag
 			</blockquote>
 		</blockquote>
 
+		<blockquote id="dev.eventref.output.menu.issue.relationship">
+			<title>EVENT_MENU_ISSUE_RELATIONSHIP (Default)</title>
+			<blockquote>
+				<para>
+					This event allows plugins to add new links to the menu
+					under the <emphasis>Relationships</emphasis> section
+					in the View Issue Details page.
+				</para>
+
+				<itemizedlist>
+					<title>Parameters</title>
+					<listitem><para>&lt;Integer&gt;: bug ID</para></listitem>
+				</itemizedlist>
+
+				<itemizedlist>
+					<title>Return Value</title>
+					<listitem>
+						<para>&lt;Array&gt;: List of elements to add to the menu.</para>
+						<para>This is an associative array, with the link's label as key
```

Comment:
It would be good to have the event expect the result as an array of elements.  Each element has a `label`, `url`, and possibly more can be added in the future.

Reply:
That was my initial thought for the implementation, but then I settled on this method, to be consistent with EVENT_MENU_ISSUE.

If we're going the way of associative array, I'm wondering if we should not take it one step further, and define Classes for menu items... This may also help addressing the problem I [mentioned above](https://github.com/mantisbt/mantisbt/pull/1567#discussion_r326889691). 

Thoughts ?

Reply:
I agree that classes is the way to go.  It is unfortunate that we will end up with 3 models though (array, assoc array, and classes).  But I think discoverability and functionality will be much better if we have a model where each event has XyzEventArgs for the arguments and potentially functions that can be retrieved by the plugin.  I think we should also consider whether we should have XyzEventResult.

Reply:
Follow-up in https://github.com/mantisbt/mantisbt/pull/1567#issuecomment-561543366